build.sbt

```diff
@@ -4,6 +4,10 @@ name := "sanity-tests"
 
 version := "1.0"
 
+scalaVersion := "2.11.5"
+
+scalacOptions ++= Seq("-feature")
```

Comment:
It was to debug a feature warning. But I think it's useful to leave it in there. I always enable this flag for Scala projects. 

Without it, you get compiler warnings saying "be careful, you used a smarty pants feature of Scala" but without any useful info about why the warning was generated.